### PR TITLE
sci-visualization/gcalc: EAPI bump 4 -> 6

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

**Comment by jlec** [comment]
Please drop `die` after newicon.

File: sci-visualization/gcalc/gcalc-3.1_rc1-r1.ebuild [hunk 1]
```diff
@@ -22,8 +22,9 @@ DEPEND=">=virtual/jdk-1.4"
 
 S="${WORKDIR}/${MY_P}"
 
-java_prepare() {
+src_prepare() {
 	find -name '*.jar' -exec rm -v {} + || die
```
Comment (SoapGentoo) [inline sci-visualization/gcalc/gcalc-3.1_rc1-r1.ebuild]: prefer GNUism `-delete` over `-exec rm -v {} +`

Comment (chewi) [inline sci-visualization/gcalc/gcalc-3.1_rc1-r1.ebuild]: True, though there's also `java-pkg_clean`.

Comment (SoapGentoo) [inline sci-visualization/gcalc/gcalc-3.1_rc1-r1.ebuild]: Even better, use the right tools for the job, use `java-pkg_clean` then

**Comment by chewi** [comment]
Sorry, didn't have a chance to look at the whole thing before. This would actually be much better if it used java-pkg-simple.eclass. You'd need to use `java-pkg_addres` to add the resources. See [jackson-2.8.5.ebuild](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/jackson/jackson-2.8.5.ebuild) for a good example. Also depend on at least the lowest Java version we have in the tree (1.7).

**Comment by g-braeunlich** [comment]
@chewi
Could you have a look, if my result is somehow reasonable?

**Comment by chewi** [comment]
Merged, thanks! `java-pkg-2_src_prepare` could have been added and you don't need to set `IUSE` when it's empty but those aren't a big deal.

**Pull request closed without merging**